examples/grenade-examples.cabal

```diff
@@ -39,37 +37,57 @@ executable             mnist
     ghc-options:       -Wall -threaded -O2
     main-is:           main/mnist.hs
     build-depends:     base
+                     , cereal
                      , grenade
-                     , attoparsec
-                     , either
+                     , filepath
+                     , bytestring
                      , optparse-applicative            >= 0.13       && < 0.16
-                     , text                            == 1.2.*
                      , mtl                             >= 2.2.1      && < 2.3
                      , hmatrix                         >= 0.18       && < 0.21
                      , transformers
                      , semigroups
                      , singletons
                      , MonadRandom
                      , vector
+                     , split
+                     , zlib
+
+executable             iris
```

Comment:
A new example is a great idea, but should probably be a separate PR from the other changes you are making.

Having them a separate make reviewing both PRs easier.

Reply:
Understood.  In this case the changes are focused on making the examples run (which they now do, as you can see from the travis output).  I think that I've got to a stage where the pieces are in place.  If you want to improve the run scripts, I think that that would be good.  I'd be happy to flesh out some of the documentation, if that would help.

Reply:
Not too sure it the examples should run in CI. Will have a look.